html/full-demo.html

```diff
@@ -30,9 +30,13 @@
 
 <!-- Set to http://localhost:8080/gxa/ or http://localhost:8080/gxa/sc/ -- Remember the trailing slash! -->
 <script>
+    const symbol = `foobar`;
+
     fetchLoaderDemo.render({
       host: 'http://localhost:8080/gxa/sc/',
-      resource: 'json/search?symbol=BRCA1'
+      resource: 'json/search?symbol=' + symbol,
+      resultsMessageFormatter: function(data) { return symbol + ' ' + data.searchExplanation + ' is expressed in:'; },
```

Comment:
I am not sure it is good to put `data.searchExplanation` as a prop because I remember one story is about appending the `geneID` behind the queried `geneSymbol` if and only if all the expressed results belong to a single `geneID`. Where shall we put the condition in this case?

Reply:
In the back end. If the matched gene IDs are more than one or the same as the queried string, `data.searchExplanation` will be empty.